cmd/sink/routers/result.go

```diff
@@ -0,0 +1,19 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package routers
+
+import (
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/protocol"
+)
+
+func NewCEResult(statusCode int, err error) protocol.Result {
+	// Cloud Event Processed successfully
```

Comment:
I think this comment belongs inside the `true` clause of the `if` statement.

Reply:
you're right. Moved it inside `if`